src/interface/src/app/types/legend.types.ts

```diff
@@ -1,4 +1,4 @@
-export const DEFAULT_COLORMAP = 'viridis';
+export const DEFAULT_COLORMAP = 'turbo';
```

Comment:
just checking my understanding: colormap is not actually used as a param for getting the rasters anymore, right?

Reply:
Yes, that's right!